doc/release-process.md

```diff
@@ -46,12 +47,12 @@ To release a new version of Weave Gitops, you need to:
     staging site (there's a link in the list of github status checks
     called "Doc site preview")
 - The PR cover message contains draft release notes. Edit the cover
-  message to fill in or delete blocks as appropriate.
+  message to fill in or delete blocks as appropriate. Move as many PRs out of "Uncategorized" as you possibly can. 
 - If everything looks good, approve the PR - do *not* merge or things
   won't be published in the right order. This immediately kicks off the
   release job.
-- Wait for the action to finish (~20 minutes), at which point the PR
-  will be merged automatically.
+- Cross you fingers and ask for a blessing from Mr. Kubernetes, then wait for the action to finish (~20 minutes), at which point the PR will be merged automatically.
```

Comment:
AFAIR, in the release action only the build-and-push-image step takes over an hour.

Reply:
And I think it's better not to specify time even approximately because everything  can change after modifying the release process. It will not be easy to remember to check the new time and update the docs in this particular place.